web/app/widgets/chart/chart.widget.js

```diff
@@ -183,6 +187,56 @@
                     vm.interactiveChartOptions.series.push(seriesoptions);
                 }
 
+                // if set to true datasets which don't receive an update for a long time 
+                // will have their dataset filled with latest state when other datasets receive an update
+                var fillValues = true;
+                // if set to true live updates are supported
+                var supportLiveUpdates = true;
```

Comment:
Should it be possible to change the behaviour behind this switch by adding a configuration option to the config dialog or should it be activate by default?